packs/pagerduty/actions/lib/action.py

```diff
@@ -0,0 +1,31 @@
+from st2actions.runners.pythonrunner import Action
+import pygerduty
+
+
+
+class PagerDutyAction(Action):
+  def __init__(self, config):
+    super(PagerDutyAction, self).__init__(config)
+    self.pager = self._init_client()
+    self.trigger = []
+  
+  def _init_client(self):
+    api_key = self.config['api_key']
+    service_api = self.config['service_api']
+    subdomain = self.config['subdomain']
+    pager = pygerduty.PagerDuty(subdomain, api_token=api_key)
+    return pager
+
+##get all the acknowledged incidents
+  def ack_incidents(self):
```

Comment:
To make it more explicit that this and the method bellow returns stuff, please add `get_` prefix (e.g. `get_ack_incidents`).